pathetic-model/src/main/java/org/patheloper/util/ChunkUtils.java

```diff
@@ -47,4 +48,14 @@ public Material getMaterial(ChunkSnapshot snapshot, int x, int y, int z) {
         return snapshot.getBlockType(x, y, z);
     }
 
+    /**
+     * Get the block state from a chunk snapshot at the given coordinates
+     */
+    public BlockState getBlockState(ChunkSnapshot snapshot, int x, int y, int z) {
+        if (BukkitVersionUtil.getVersion().isUnder(13, 0))
```

Comment:
Seems like this could be optimised. An int would be nice rather than major minor seperately